README.md

```diff
@@ -63,22 +63,116 @@ In this dialog, options should be filled out as follows:
+mobie administration interface. The table contains one row for every mobie identity and the value for the associated mobie user's security associations in the columns for that row. If there are multiple identities
+for a single mobie user, there will be a row for each identity with the security association values duplicated for each identity.
+
+This table can be used to implement dynamic row level security in your Power BI visualization. For example, you may wish to restrict a user to seeing only data where a Company column matches the Company security
+association of the logged in user. To do this, use the "Manage relationships" dialog available from the Modeling tab in Power BI. In this dialog, create a new relationship with the "Security Associations" table
+and the desired data table. Select the Company column in both and pick the "Cross filter direction" to be "Single" from the "Security Associations" table to the data table. Note that the cardinality of this
+relationship will be many-to-many and you will want to make sure that the relationship is active.
+
+Next, you will want to "Manage roles" (also from the Modeling tab) in Power BI. Here you will want to create a new role with the DAX expression of "[Identity] = USERPRINCIPALNAME()". You might call this role
+"Company Security". Now, when you publish your visualization, you also want to assign this role to any users that you publish the visualization to. In this way, it will look up their user principal name
+and match it up with the Identity column in the Security Associations table. Because there is a cross filtering relationship between the Security Associations table and your data table, the contents of the
+data table will also be filtered. Note that specifying User Principal Names (UPN) is only available in Data Access Studio beginning with build 8.0.11.1.
+
+## Release Notes
+
+### Release 1.0.0.3
+
+* **Breaking change**: Navigation names used by the connector now include the \<Report Owner\> as part of the comma separated name to distinguish between versions of a report name owned by different users. Published reports use the name DAS_PUBLIC for the report owner. When first opening existing Power BI files with connections to mobie snapshots prior to this release, edit the query and include the report owner name in the comma separated name.
+* Security Associations table included as a mechanism to implement dynamic row level security in your Power BI report.
+* Support ability to connect to FDASPROPT test table.
+* Improve error checking of incoming options and make parameters optional that are not always required.
+* By default, users only see snapshots that they have published themselves to the mobie DataBox. To retain the past behavior of seeing all snapshots, it is necessary to create a table called rn__Options. See below for instructions.
```

Comment:
It should be made clear that this table is not a requirement and only needed for this legacy behavior.

Reply:
OK. Added a note.